packages/destination-actions/src/destinations/fullstory/__tests__/fullstory.test.ts

```diff
@@ -67,7 +67,7 @@ describe('FullStory', () => {
         event: {
           event_name: eventName,
           event_data: {
-            'first-property_str': properties['first-property'],
+            firstproperty_str: properties['first-property'],
```

Comment:
This assertion was no longer valid; we used to leave property names unchanged on `track` calls.